firefox-ios/Client/Application/AppDelegate.swift

```diff
@@ -164,8 +164,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
         // Cleanup can be a heavy operation, take it out of the startup path. Instead check after a few seconds.
         DispatchQueue.main.asyncAfter(deadline: .now() + 5.0) { [weak self] in
-            // TODO: testing to see if this fixes https://mozilla-hub.atlassian.net/browse/FXIOS-7632
-            // self?.profile.cleanupHistoryIfNeeded()
+            self?.profile.cleanupHistoryIfNeeded()
```

Comment:
I thought we were doing a feature flag for this?

Reply:
Updated! It was not done yet haha